pkg/npm/npm_test.go

```diff
@@ -33,26 +33,33 @@ func TestNpm(t *testing.T) {
 			Options: options,
 		}
 
-		packageJSONFiles := exec.FindPackageJSONFiles()
+		packageJSONFiles, err := exec.FindPackageJSONFiles(nil)
 
-		assert.Equal(t, []string{"package.json"}, packageJSONFiles)
+		if assert.NoError(t, err) {
+			assert.Equal(t, []string{"package.json"}, packageJSONFiles)
+		}
 	})
 
 	t.Run("find package.json files with two package.json and filtered package.json", func(t *testing.T) {
 		utils := newNpmMockUtilsBundle()
 		utils.AddFile("package.json", []byte("{}"))
 		utils.AddFile(filepath.Join("src", "package.json"), []byte("{}"))
 		utils.AddFile(filepath.Join("node_modules", "package.json"), []byte("{}")) // is filtered out
+		utils.AddFile(filepath.Join("gen", "package.json"), []byte("{}"))          // is filtered out
+		utils.AddFile(filepath.Join("filter", "package.json"), []byte("{}"))       // is filtered out
+		utils.AddFile(filepath.Join("filterPath", "package.json"), []byte("{}"))   // is filtered out
```

Comment:
I would add more test cases which check that the matching works. like 
```suggestion
		utils.AddFile(filepath.Join("filterPath", "package.json"), []byte("{}"))   // is filtered out
		utils.AddFile(filepath.Join("notfiltered", "package.json"), []byte("{}"))   // should NOT be filtered out
```

Maybe also more nested paths to check edge cases. I mean... try to break it! :-)

Reply:
With the nested ones you are right, but for unfiltered one there is already the case with `src/package.json` and `package.json`

Reply:
I used "notfiltered" because it contains the word "filter" for which there is an exclude pattern. That would reveal mistakes in the filtering, when it is not done properly on separator boundaries. Sorry, should have made it clear in the first place.

Reply:
Ah I see good point 👍